### DISCUSSION TITLE
Repo Unforkable After Fork Delete, Busy, Leads to 404

### DISCUSSION THREAD

**Discussion by classicvalues** [community/community#83767] [Repositories]
I was busy looking through my @classicvalues/go fork a couple of days ago when I noticed that the fork was from a source other than @golang/go. I deleted that said fork in order to re-fork from @golang/go, but when I went to fork, it looked like I still had a fork on @classicvalues/go. I checked the webpage in order to delete it again, but it lead to a '404 webpage not found' error. 

I have not found this error on the internet again and would like some suggestions from people in the community that have had this error happen to them before and perhaps some suggestions on how I can fix these errors going forward. Hopefully this also helps anyone that might also one day go through the same problem. I have also reached out to GitHub customer support, if we find an answer I will immediately post it on this Discussion Thread. 

It might be that the "restore repository within 90 days" is in effect and I have found in the documents that the issue might involve that it is still saved within the archive of the 90 days from the initial delete. Though, within the document it also suggests that "If you want to restore a repository that was part of a fork network that is not currently empty, you can contact us through the [GitHub Support portal](https://support.github.com/)." which I have done, perhaps it is just policy and they will allow me to form a repository in the same file location under the same name. 

Here is an image of the 'Settings' => "Repositories" => "Deleted Repositories" site, where I am asked to "Contact Customer Support" to restore the repository, though I would like to completely overwrite it, I am sure that if we can restore it we can also just open it in the shell or on GitHub Desktop and remove the old files and replace them with the better and more direct @golang/go repository. 

![image](https://github.com/community/community/assets/71681815/f47c8830-4e9c-42e5-a14c-2ae6b59f97db)

Best of wished, and Happy 2024 New Year.

**Comment by afidyoga** [comment]
It seems like you are referring to an issue or topic related to a code repository on a platform like GitHub, possibly involving forking, deleting forks, and encountering a 404 error. To address this kind of issue, you might want to consider the following steps:

Clear Browser Cache:
Sometimes, cached information in your browser might be causing the issue. Clear your browser cache and try accessing the repository again.

Check Repository Status:
Ensure that the original repository is still available and hasn't been deleted or made private. If it has been deleted or made private, forking won't be possible.

Fork Again:
If the original repository is still available, try forking it again. Make sure you are logged into your GitHub account and have the necessary permissions.

Check Fork Status:
Confirm that your fork of the repository has been created successfully. You should be able to see it in your GitHub account under the "Repositories" tab.

Repository Permissions:
Verify that you have the necessary permissions to fork and access the repository. If the repository is private, you need to be granted access by the repository owner.

404 Error:
If you're getting a 404 error, it usually means that the resource you are trying to access cannot be found. Double-check the URL to ensure it's correct. If the issue persists, it might be a temporary problem on the platform, and you can try again later.

GitHub Support:
If the problem persists and none of the above steps resolve the issue, you may want to reach out to GitHub support for assistance. They can investigate further and help you with any technical issues related to the platform.

Remember, the exact steps may vary depending on the specifics of your situation, and the suggestions provided are based on common issues related to repository forking and 404 errors on GitHub.
